src/openrct2/peep/Peep.h

```diff
@@ -775,6 +776,12 @@ struct Guest : Peep
     bool DecideAndBuyItem(Ride* ride, int32_t shopItem, money32 price);
     void SetSpriteType(PeepSpriteType new_sprite_type);
 
+    void HandleEasterEggName();
+    int32_t GetEasterEggNameId();
+    void UpdateEasterEggInteractions();
```

Comment:
Three blank lines are two too many.
This comment applies to the whole PR - I won't spam this comment ;-)